src/lib/user/user-pii.ts

```diff
@@ -0,0 +1,49 @@
+import { EncryptionService } from "../services/encryption";
+
+export type UserPii = {
+  userName: string;
+  userEmail: string;
+  userPhoneNumber: string;
+};
+
+export type DogOii = {
+  dogName: string;
+};
+
+export async function encryptUserPii(
+  pii: UserPii,
+  encryptionService: EncryptionService,
+): Promise<string> {
+  const data = JSON.stringify(pii);
```

Comment:
Use JSON schema.

Reply:
We can add this retrospectively. It will work with existing encoded JSON strings. Thus, we will defer this to another ticket.